### PR TITLE
Fix issue where Carthage doesn’t build for watchOS or tvOS if bitcode is disabled in Xcode 14

### LINKED ISSUE #3292: In Xcode 14, Carthage does not create tvOS & watchOS builds when Bitcode is disabled
* carthage install method: [ ] .pkg, [x] homebrew, [ ] source
* `which carthage`: `/opt/homebrew/bin/carthage`
* `carthage version`: 0.38.0
* `xcodebuild -version`: Xcode 14.0 - Build version 14A5270f
* Are you using `--no-build`? No
* Are you using `--no-use-binaries`? No
* Are you using `--use-submodules`? No
* Are you using `--cache-builds`? No
* Are you using `--new-resolver`? No
* Are you using `--use-xcframeworks`? Yes

**Actual outcome**
The final XCFramework produced by Carthage contains a `tvos-arm64_x86_64-simulator` folder, but not a `tvos-arm64` folder. This is occurring because Bitcode is disabled by default on tvOS targets in Xcode 14, but Carthage [filters out the `appletvos` sdk](https://github.com/Carthage/Carthage/blob/master/Source/CarthageKit/Xcode.swift#L696-L701) if Bitcode is disable because it assumes Bitcode is required for Apple TV. This is no longer true in Xcode 14.

**Expected outcome**
Carthage should still build products for the `appletvos` and `watchos` sdks when Bitcode is disabled and Xcode 14 is selected.

### PR DESCRIPTION
**Pull request by daltonclaybrook**
Resolves #3292

### CODE REVIEW

**Comment by SSheldon** [comment]
This fix resolved our watchOS build issue on Xcode 14, thank you!

**Comment by lukegardiner** [comment]
Can this get released soon now that Xcode 14 is officially out and auto-updating through the Mac App Store?

**Comment by daltonclaybrook** [comment]
@tmspzz Any thoughts on merging this?

**Comment by StephanWalters** [comment]
@tmspzz can this be merged?

**Comment by RushanB** [comment]
Can someone merge this in?

**Comment by daltonclaybrook** [comment]
@RushanB I would if I could. 😅 In the meantime, you can check out this branch and run `make install`.

**Comment by daltonclaybrook** [comment]
@jshier Hey Jon, sorry to tag you. Do you see any impediment to merging this PR?

**Comment by jshier** [comment]
@daltonclaybrook No, but then I'm not a project maintainer here, so I have no merge privileges. Sorry.

**Comment by daltonclaybrook** [comment]
@jshier Ah whoops, saw you were the author of a recently-merged commit and assumed. Sorry about that!

**Comment by ttuygun** [comment]
Mentioned project members commits actively to another projects in Github. For this reason, it seems they abandoned the project. We need a ReCarthage, the re-born of Carthage with active community😀

> @RushanB I would if I could. 😅 In the meantime, you can check out this branch and run `make install`.

**Comment by jvaladez** [comment]
@erichoracek or @giginet could we get your help on this?

**Review by giginet — APPROVED** [review]
Looks good to me

**Comment by giginet** [comment]
I think this PR makes sense. I'll merge this soon.

**Comment by LuisFelipe635** [comment]
Hi guys!
When this fix will be released? I am facing big troubles to migrate an iOS project to Xcode 14 and this could help a lot

**Comment by daltonclaybrook** [comment]
@LuisFelipe635 I can't comment on when it will be released, but in the meantime, you can clone the Carthage repo and run `make install` to build a version of Carthage with this fix.

**Comment by LuisFelipe635** [comment]
Thanks, @daltonclaybrook, I'll try that

**Comment by gemmakbarlow** [comment]
@giginet - would you have an estimate of when this might be released ?

**Comment by giginet** [comment]
I'm not a core maintainer. So I haven't released Carthage, So I relied on other maintainers.
However, this project seems to be abandoned...
I'm going to try to release a newer version.

**Comment by giginet** [comment]
I asked core maintainers to release a new version. However, anybody no longer exists....

I released 0.39.0 now. https://github.com/Carthage/Carthage/releases/tag/0.39.0
Sorry for waiting you.

**Comment by gemmakbarlow** [comment]
> I asked core maintainers to release a new version. However, anybody no longer exists....
> 
> 
> 
> I released 0.39.0 now. https://github.com/Carthage/Carthage/releases/tag/0.39.0
> 
> Sorry for waiting you.

Nice work, and a big thank-you !

File: Source/CarthageKit/Xcode.swift [hunk 1]
```diff
@@ -676,6 +676,9 @@ public func buildScheme( // swiftlint:disable:this function_body_length cyclomat
 	sdkFilter: @escaping SDKFilterCallback = { sdks, _, _, _ in .success(sdks) }
```
[the remark this answers is not in the document]
Comment (tharkhay) [reply Source/CarthageKit/Xcode.swift:676]: > Coming 
- [ ]

**Review by tharkhay — COMMENTED** [review]
[]()

**Comment by joshuarendora1** [comment]
Hi Good Day before we begin can i ask a little favor to Subscribe these
Youtube chanel?

Please click Link Below. 👇👇


https://youtube.com/shorts/va3bZK4wiKU?feature=share



YOU'RE SUBSCRIPTION IS REALLY A BIG HELP TO THE CHARITY WE CHOSEN TO
RECEIVED 75% OF THE SALLARY WE MAKE IN THE FUTURE.

WE ARE EXPECTING THAT YOU ARE ALSO IN LINE AND 1 OF THOSE PEOPLE ALSO WHO
SUPPORT WHAT WE MAKE AND PLAN IN THE FUTURE IN HELPING OTHERS ♥️♥️♥️

THANK YOU SO MUCH ♥️

**Pull request merged**